app/src/main/res/layout/sheet_blood_sugar_entry.xml

```diff
@@ -17,8 +20,7 @@
     android:outAnimation="@anim/measurementinput_reading_exit_to_left"
     android:paddingTop="@dimen/spacing_12"
     android:paddingBottom="@dimen/spacing_8"
-    app:debug_displayedChild="0"
-    tools:background="@color/white">
+    app:debug_displayedChild="1">
```

Comment:
Did we changed the default displayed child in the layout editor for a reason?

Reply:
Ah, I was just trying to verify the change and must have forgotten to revert it back. I just want to check, will it impact which page app displays or is just for development purposes?

Reply:
It's only for development purposes, just wanted to clarify why it happened.